### ISSUE TITLE
No compatibility with Efinity 2024.2.294

### ISSUE THREAD

**Issue by m-byte** [enjoy-digital/litex#2126]
There are multiple changes in 2024.2 that seem to break compatibility:
1. the `pt/db/device_*.xml` files are gone. This might be intentional, however, I just copied them over from the previous version for myself, which seems to help (will investigate further at some later point in time)
~~2. the vdb file is placed in the `outflow` directory, while litex expects it to be in `work_syn`. As a workaround, I added a check into my local codebase, whether the file exists in `work_syn` and use the `outflow` directory if it doesn't.~~

**Comment by enjoy-digital** [comment]
Thanks @m-byte, we can probably close this since your PR has been merged.

**Comment by m-byte** [comment]
@enjoy-digital not quite, as I only took care of the secind point above.

**Comment by enjoy-digital** [comment]
@m-byte: OK thanks, that's re-opened then.

**Comment by jwise** [comment]
Good idea on copying over the xml file from the previous version :) I looked briefly to see if the name had changed, and it appears that not just the name has changed but also the structural content.

**Comment by jwise** [comment]
Thanks so much, @m-byte!
